convert2rhel/pkgmanager/__init__.py

```diff
@@ -16,14 +16,21 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 try:
+    # this is used in pkghandler.py to parse version strings in the _parse_pkg_with_yum function
+    from rpmUtils.miscutils import splitFilename
     from yum import *
 
     # This is added here to prevent a generic try-except in the
     # `check_package_updates()` function.
     from yum.Errors import RepoError  # lgtm[py/unused-import]
 
     TYPE = "yum"
+
+# WARNING: if there is a bug in the yum import section, we might try to import dnf incorrectly
```

Comment:
Not a bug, per say, more like "try to import dnf if we fail to import yum on RHEL8+, which is likely to happen"

Reply:
Wrote this comment because during development I had an incorrect yum import which caused dnf to get imported when we really wanted yum.

Reply:
Ah, I see. Now it makes sense to me. Will close the conversation as I think this is a valid comment.